apps/cli/src/commands/appUninstallAll.ts

```diff
@@ -16,20 +18,27 @@ export default {
   }>) =>
     withDevice(device || "")(t => {
       const exec = execWithTransport(t);
-      return from(getDeviceInfo(t)).pipe(
-        mergeMap(deviceInfo =>
-          listApps(t, deviceInfo).pipe(
-            filter(e => e.type === "result"),
-            map(
-              (e: any) =>
-                reducer(e.result, {
-                  type: "wipe",
-                }),
-              exec,
-            ),
-            mergeMap(s => runAll(s, exec)),
-          ),
-        ),
+      return from(t.exchange(Buffer.from("e0dd000000", "hex"))).pipe(
```

Comment:
It's ok to have a "raw" APDU here ? I don't find it on the list of known apdu, could you add a comment above it pls 🙏

Reply:
It's actually stupid because it should be using the command instead which is the whole point, good catch